test/test_flow_control_window.py

```diff
@@ -52,6 +52,28 @@ def test_flow_control_decreases_with_sent_data(self):
         remaining_length = self.DEFAULT_FLOW_WINDOW - len(b'some data')
         assert (c.local_flow_control_window(1) == remaining_length)
 
+    def test_flow_control_decreases_with_sent_data_with_padding(self):
+        """
+        When data is sent on a stream, the flow control window should drop.
```

Comment:
Do you mind updating this docstring to be unique and to explain specifically what it tests? That is, that the flow control window drops by an amount that includes the padding?